Document cron (#167)
iov-one/docs: docs/weave/tutorial/09-app.md

- `cash.NewFeeDecorator`: returns a static fee decorator with the given minimum fee, and all collected fees going to a default address
- `batch.NewDecorator`: returns a batch transanction decorator

### CRON stack

> We will dive in to CRON stack and it's use in the next chapters, but there is no harm in giving it a peek.

Reviewer: ```suggestion
We will dive into CRON stack and its use in the next chapters, but there is no harm in giving it a peek.
```

> Cron stack is a minimal application that runs differently from the main decorators that runs scheduled jobs in the background.

Author: :+1:
